repose-aggregator/tests/performance-tests/roles/gatling/tasks/install_gatling.yml

```diff
@@ -49,7 +49,7 @@
     state: absent
 
 - name: Copy Gatling configuration
-  copy:
+  template:
```

Comment:
I like that it uses the section (e.g. copy, template) to determine where the file should be. That way you could copy a template to be interpreted by a different process later. Someone was using their noodle.